Akash/k8s service dotnet sdk

akashkeshari opened the pull request:
# All SDK Contribution checklist:

This checklist is used to make sure that common guidelines for a pull request are followed.
- [ ] **Please open PR in `Draft` mode if it is:**
	- Work in progress or not intended to be merged.
	- Encountering multiple pipeline failures and working on fixes.
- [ ] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md).**
- [ ] **The pull request does not introduce [breaking changes](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/breaking-change-rules.md).**

### [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md#general-guidelines)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md#testing-guidelines)
- [ ] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md#sdk-generation-guidelines)
- [ ] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code. (Track 2 only)
- [ ] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK. Please double check nuget.org current release version.

## Additional management plane SDK specific contribution checklist: 
Note: Only applies to `Microsoft.Azure.Management.[RP]` or `Azure.ResourceManager.[RP]`
 
- [ ] Include updated [management metadata](https://github.com/Azure/azure-sdk-for-net/tree/master/eng/mgmt/mgmtmetadata).
- [ ] Update AzureRP.props to add/remove version info to maintain up to date API versions.

### Management plane SDK Troubleshooting
- If this is very first SDK for a services and you are adding new service folders directly under /SDK, please add `new service` label and/or contact assigned reviewer.
- If the check fails at the `Verify Code Generation` step, please ensure:
	- Do not modify any code in generated folders.
	- Do not selectively include/remove generated files in the PR.
	- Do use `generate.ps1/cmd` to generate this PR instead of calling `autorest` directly.
	Please pay attention to the @microsoft.csharp version output after running generate.ps1. If it is lower than current released version (2.3.82), please run it again as it should pull down the latest version,

### Old outstanding PR cleanup
 Please note:
	If PRs (including draft) has been out for more than 60 days and there are no responses from our query or followups, they will be closed to maintain a concise list for our reviewers.

Commit:
added generated clients

Commit:
added csproj files

Commit:
added mgmt text file

Commit:
Merge branch 'master' into akash/k8s_service_dotnet_sdk

Commit:
added generated models from updated swagger

Commit:
added tests

Commit:
added record sessions

markcowl commented:
swagger PR is here: https://github.com/Azure/azure-rest-api-specs/pull/13015

markcowl commented on sdk/hybridkubernetes/Microsoft.Azure.Management.Kubernetes/src/Microsoft.Azure.Management.Kubernetes.csproj:
```diff
@@ -0,0 +1,27 @@
+    <Description>
+      Provides developers with libraries for the hybrid kubernetes platform under Azure Resource manager to deploy Connected Clusters resource. Note: This client library is for Connected Clusters under Azure Resource Manager.
+    </Description>
+    <Version>1.0.0</Version>
```
The first version should be ```1.0.0-beta.1```

akashkeshari replied:
Updated in the latest commit

markcowl reviewed (changes requested):
A few additions are needed:
- Add AssemblyInfo.cs in the Properties directory, similar to the one here: 
- Add your project to the management ci job before this line:  https://github.com/Azure/azure-sdk-for-net/blob/master/eng/pipelines/mgmt.yml#L79  this will enable ci runs to build your project and run your tests

Commit:
added assembly info file

akashkeshari commented:
> A few additions are needed:
> 
> * Add AssemblyInfo.cs in the Properties directory, similar to the one here:
> * Add your project to the management ci job before this line:  https://github.com/Azure/azure-sdk-for-net/blob/master/eng/pipelines/mgmt.yml#L79  this will enable ci runs to build your project and run your tests

1. Added assemblyinfo.cs
2. updated mgmt.yml

Please review again
